[2025-01-23T13:53:09.798+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-23T13:53:09.809+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_cr_table_snowflake.print_message manual__2025-01-23T13:52:56.388665+00:00 [queued]>
[2025-01-23T13:53:09.815+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_cr_table_snowflake.print_message manual__2025-01-23T13:52:56.388665+00:00 [queued]>
[2025-01-23T13:53:09.815+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-23T13:53:09.823+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): print_message> on 2025-01-23 13:52:56.388665+00:00
[2025-01-23T13:53:09.829+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=9556) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-23T13:53:09.830+0000] {standard_task_runner.py:72} INFO - Started process 9557 to run task
[2025-01-23T13:53:09.831+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_cr_table_snowflake', 'print_message', 'manual__2025-01-23T13:52:56.388665+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/create_table.py', '--cfg-path', '/tmp/tmp_lcmssq1']
[2025-01-23T13:53:09.833+0000] {standard_task_runner.py:105} INFO - Job 13: Subtask print_message
[2025-01-23T13:53:09.867+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_cr_table_snowflake.print_message manual__2025-01-23T13:52:56.388665+00:00 [running]> on host c28716311b4b
[2025-01-23T13:53:09.919+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_cr_table_snowflake' AIRFLOW_CTX_TASK_ID='print_message' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T13:52:56.388665+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-23T13:52:56.388665+00:00'
[2025-01-23T13:53:09.920+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-23T13:53:09.922+0000] {logging_mixin.py:190} INFO - Snowflake query execution is complete!
[2025-01-23T13:53:09.922+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-23T13:53:09.931+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-23T13:53:09.931+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_cr_table_snowflake, task_id=print_message, run_id=manual__2025-01-23T13:52:56.388665+00:00, execution_date=20250123T135256, start_date=20250123T135309, end_date=20250123T135309
[2025-01-23T13:53:09.967+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-23T13:53:09.978+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-23T13:53:09.984+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
